core/integration/bookkeeping/client.go

```diff
@@ -119,6 +120,7 @@ func NewClient(cxt context.Context, cancel context.CancelFunc, endpoint string)
 		FlpServiceClient:         bkpb.NewFlpServiceClient(conn),
 		LogServiceClient:         bkpb.NewLogServiceClient(conn),
 		RunServiceClient:         bkpb.NewRunServiceClient(conn),
+		LhcFillServiceClient:     bkpb.NewLhcFillServiceClient(conn),
```

Comment:
Please add the upstream protofile URL to the AliECS Makefile, so `make vendor` can pull it automatically as it does for other Bookkeeping protofiles.